ci_framework/roles/build_containers/tasks/hotfix.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Run tcib hotfix command
+  ansible.builtin.command: >
+    openstack tcib container image hotfix
+    {% for image, rpms_path, tag in images|zip(rpms_paths, tags) %}
```

Comment:
afaik this will keep an empty line  - you probably want to use `-%}` in order to remove it.

Reply:
done